recipes/pyem/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pyem" %}
+{% set version = "0.62" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/asarnow/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a3e97c2ca83229617b04babf8912ef11e9258b225b2c3e559d654886bdc32d16
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win or py<39 or py>311]
+  entry_points:
+    - cfsc.py = pyem.cli.cfsc:_main_
+    - csparc2star.py = pyem.cli.csparc2star:_main_
+    - ctf2star.py = pyem.cli.ctf2star:_main_
+    - disparticle.py = pyem.cli.disparticle:_main_
+    - emcalc.py = pyem.cli.emcalc:_main_
+    - map.py = pyem.cli.map:_main_
```

Comment:
Some of these entry points are a bit too generic and cause conflicts with other tools (which also made the same poor choice). Usually a project specific prefix is desired; e.g. `pyem-map`. But I'll leave that to upstream.